src/bin/flamegraph.rs

```diff
@@ -18,21 +21,21 @@ impl Into<Options> for Opt {
 }
 
 fn main() -> quick_xml::Result<()> {
-    let (infile, options) = {
+    let (infiles, options) = {
         let opt = Opt::from_args();
-        (opt.infile.clone(), opt.into())
+        let infiles = opt
+            .infiles
+            .iter()
+            .map(|f| File::open(f).map_err(quick_xml::Error::Io))
```

Comment:
I wonder if we can special-case `-` here somehow so that I could do:
```console
$ inferno-flamegraph foo.folded -
```

Reply:
Do you want to be able to handle `-` with files also passed in, or only by itself?

Reply:
I just pushed a commit that handles `-` by itself.

Reply:
I was thinking specifically when it is combined with other files. That might be tricky to pull off though sadly, since the `Read` is a different type. We could get around this by `Box`ing the `Read` we pass to `from_readers`, but that's a little sad too. Handling `-` on its own I don't think is worth it.

Reply:
OK, I got that working.